tests/domain/beneficiary_bookings/beneficiary_booking_stock_test.py

```diff
@@ -40,3 +40,110 @@ def should_returns_false_when_stock_has_future_booking_limit_datetime(self):
 
             # When / Then
             assert stock.has_booking_limit_datetime_passed is False
+
+    class IsAvailableForBookingTest:
+        def should_return_false_when_booking_limit_datetime_has_passed(self):
```

Comment:
Des noms de tests plus métiers !
```suggestion
        def should_not_be_available_when_booking_limit_datetime_has_passed(self):
```